lib/Bootstrap.php

```diff
@@ -123,14 +123,14 @@ public static function boostrap()
         $loader = include __DIR__ . '/../../../../vendor/autoload.php';
         self::defineConstants();
 
-        if (is_integer(PIMCORE_PHP_ERROR_REPORTING)) {
+        if (defined('PIMCORE_PHP_ERROR_REPORTING')) {
```

Comment:
This check is useless, the constant is always defined ;-) 
So I guess we can just remove the `if` condition entirely.

Reply:
Less code is even better! 🎉